fiware-region-sanity-tests/commons/constants.py

```diff
@@ -42,15 +42,20 @@
 # TASK TIMERS AND TIMEOUTS
 DEFAULT_REQUEST_TIMEOUT = 60
 MAX_WAIT_ITERATIONS = 60
+MAX_WAIT_SSH_CONNECT_ITERATIONS = 35
 SLEEP_TIME = 5
 
 # TEST DATA
 BASE_IMAGE_NAME = "CentOS-6.3init"
 TEST_FLAVOR_DEFAULT = "small"
-TEST_CIDR_PATTERN = "10.250.254.%d/24"
+TEST_CIDR_PATTERN = "10.250.%d.0/24"
 TEST_CIDR_DEFAULT = TEST_CIDR_PATTERN % 0
 TEST_SEC_GROUP_PREFIX = "testing_sec_group"
 TEST_KEYPAIR_PREFIX = "testing_keypair"
 TEST_SERVER_PREFIX = "testing_instance"
 TEST_NETWORK_PREFIX = "testing_network"
 TEST_ROUTER_PREFIX = "testing_router"
+
+# SSH CONNECTION
+SSH_CONNECTION_PORT = 22
+SSH_CONNECTION_TIMEOUT = 8 #seconds
```

Comment:
I'd suggest removing this line comment, and adding "seconds" to the general comment  above, explaining that ALL constants regarding timeouts are expressed in seconds (not in millis).